test/integration/scheduler_perf/README.md

```diff
@@ -68,6 +68,25 @@ To produce a cpu profile:
 make test-integration WHAT=./test/integration/scheduler_perf KUBE_TIMEOUT="-timeout=3600s" KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -benchtime=1ns -bench=BenchmarkPerfScheduling -cpuprofile ~/cpu-profile.out"
 ```
 
+### How to configure bechmark tests
+
+Configuration file located under config/performance-config.yaml contains a list of templates.
+Each template allows to set:
+- node manifest
+- manifests for an initial and testing pod
```

Comment:
nit: `an initial and testing pod` -> `initial and testing pods`

Reply:
Done